sails/config/jwt.js

```diff
@@ -1,5 +1,5 @@
 module.exports.jwts = {
     ACCESS_TOKEN_SECRET: "10b52b9f100d7a0647465b5de99a61a032da9c87457dca09c021a94960fdbb4ee375e5a5713190dbc6ac721c72718bcab27cbaa15fe11d29380f0dfcaf5d5eb7",
     REFRESH_TOKEN_SECRET: "fed71f70762bf1111d345956e0b078b2e161a13a66dad862b72b88b32f8466dd40e68294cd61eb126605fcb5ce22d7c9589ae265c9af89c9723a981ca6f78778",
-    EXPIRATION_TIME: "15m"
-}
\ No newline at end of file
+    EXPIRATION_TIME: "20m"
+};
```

Comment:
No newline at end of file
```suggestion
};

```